includes/cancel-on-next-payment-date.php

```diff
@@ -1,9 +1,14 @@
 <?php
-// Copied from PMPro Cancel on Next Payment Date Add On:
-// https://github.com/strangerstudios/pmpro-cancel-on-next-payment-date/
-
-// Also changed prefixes from pmproconpd_ to pmproarc_conpd_ and added compatibility
-// code to each function to avoid issues.
+/*
```

Comment:
This can be removed, might have been added to the includes file by mistake. :)

Reply:
Can revert the comments back to previous one.

Reply:
Are you referring to the header to the CONPD plugin? I thought having the CONPD version may be a good idea so I just included the whole header. Can change back if you'd prefer though

Reply:
Updated